src/getcensus.ado

```diff
@@ -483,29 +486,39 @@ if "`clear'" != "" {
     clear
 }
 
-** Product
-if "`estimates'" != "" & "`product'" != "" {
-    dis "Program will derive product from 'estimates' and ignore 'product'"
 }
```

Comment:
This extra bracket was causing an issue.

Reply:
oops. thanks for fixing.